IBM/carbon-addons-iot-react
# feat(table): add single selection and associated story

**Summary**

- Add sticky selection to the table

**Change List (commits, features, bugs, etc)**

- Add single selection to table
- Extend reducer to StatefulTable, enabling single selection
- Add new single selection story
- Update Snapshots, Tests

**Related Issues**



- Closes #202 

**Acceptance Test (how to verify the PR)**

- check out the `with single selection` story, make sure the proper actions fire
- ensure there's no regressions in `with multi select and batch actions`
